src/collections/posts/comfy-sabers.md

```diff
@@ -2,35 +2,61 @@
 title: Comfy Sabers
 section: articles
 category: event
+type: tournament
 publish: 2024-04-24T19:00:00.000Z
 homepageText: Comfy Sabers Season 3 is here! This tournament is available to players ranked lower than 2000 on ScoreSaber. Event ends June 16, 2024.
 image: /static/uploads/posts/comfy-sabers/comfy-sabers-no-text.png
 showInPostListing: true
 ---
 
+<div class=container>
 <div class="image">
     <img src="/uploads/posts/comfy-sabers/banner.png" />
 </div>
+<div class="text">
+<b>Event Name:</b> Comfy Sabers - Season 3
+<br />
+<b>Event Type:</b> Tournament
+<br />
+<b>Start Date:</b> April 24, 2024
+<br />
+<b>End Date:</b> June 16, 2024
+<br />
+<b>About this Event:</b><p>This tournament was created to allow players that normally wouldn't be able to participate in Beat Saber tournaments such as World Cup to experience the thrill of participating in a tournament. As such, any players who are better than rank 2000 on ScoreSaber are not allowed to participate; anyone above can play the qualifiers in order to participate in the tournament.</p>
+<p>There will be two divisions, a lower and a higher division, to allow for a more equal playing field. The qualifier deadline is Sunday, the 12th of May.</p>
 
-**Event Name:** Comfy Sabers - Season 3
-\
-**Event Type:** Tournament
-\
-**Start Date:** April 24, 2024
-\
-**End Date:** June 16, 2024
-\
-**About this Event:** This tournament was created to allow players that normally wouldn't be able to participate in Beat Saber tournaments such as World Cup to experience the thrill of participating in a tournament. As such, any players who are better than rank 2000 on ScoreSaber are not allowed to participate; anyone above can play the qualifiers in order to participate in the tournament.
-\
-There will be two divisions, a lower and a higher division, to allow for a more equal playing field. The qualifier deadline is Sunday, the 12th of May.
+<b>Additional Information:</b> Join the <a href="https://discord.gg/DP2d9kAzga">Comfy Sabers Discord Server</a> and check the #announcements channel
 
-**Additional Information:** Join the [Comfy Sabers Discord Server](https://discord.gg/DP2d9kAzga) and check the #announcements channel
-
-[**Watch the event here**](https://www.twitch.tv/comfysabers)
+<p><b><a href="https://www.twitch.tv/comfysabers">Watch the event here</a></b></p>
+</div>
+</div>
 
 <style>
+.container {
+  display: flex;
+  gap: 20px;
+  flex-wrap: wrap;
+  justify-content: center;
+}
 .image {
-  float: left;
-  padding: 0px 20px 20px 0px;
+  max-width: 505px;
```

Comment:
You are using these styles a lot. Ideally, markdown should only require little styling.

Anyhow, that is improvable or generalizable?

Also, I think we have a container class (not sure right now). This may conflict now or in the future. I wouldn't want to rely on scoping for that I think.

Reply:
I'm using this intentionally because markdown doesn't allow for this type of specific styling as far as I'm aware - so this is straight HTML now because that seems to be more friendly for what I'm trying to achieve here. Specifically having the image on the left and the text on the right without wrapping around the image. If you look at this page (or the Beat Cancer page where I accidentally reused this poor original layout) you'll see the layout is generally poor. I came back to update this and Beat Cancer for better layout and appearance.